doc/Installation/Installing-LibreNMS.md

```diff
@@ -1,15 +1,17 @@
 source: Installation/Installing-LibreNMS.md
 # Installing LibreNMS
 
-We have some pre-built VirtualBox images you can use to get started:
+##Images
+We have some pre-built VirtualBox images you can use to get started. Please note that these images are pre-built for VirtualBox, hence they might not work with other hypervisors:
 
 [Ubuntu 16.04](http://docs.librenms.org/Installation/Ubuntu-image/)
 
 [CentOS 7](http://docs.librenms.org/Installation/CentOS-image/)
 
-If you want to install yourself then we have some new documentation which should make it easy.
+##Manuel
+If you want to install manually then we have some documentation which should make it easy.
 
-> Please note the following docs are new and may not be 100% complete, please provide feedback on your experience.
+**Please note from January 10th 2018, the minimum supported PHP version will be 5.6.4** [More]
```

Comment:
This needs to be in all the other install docs as well as most likely people will skip straight to those. We also don't need to say from Jan the 10th here and just say, min supported version is 5.6.4 now.

Reply:
Agreed, much clearer if we just say the minimum version is 5.6.4 for newcomers.